examples/green_screen/MultiDeviceCapturer.h

```diff
@@ -4,13 +4,26 @@
 
 #include <chrono>
 #include <vector>
+#include <iomanip> // std::setw
 #include <k4a/k4a.hpp>
 
 // This is the maximum difference between when we expected an image's timestamp to be and when it actually occurred.
 // TODO waiting on a firmware update to be returned to 50
 constexpr std::chrono::microseconds MAX_ALLOWABLE_TIME_OFFSET_ERROR_FOR_IMAGE_TIMESTAMP(33000);
 // constexpr std::chrono::microseconds MAX_ALLOWABLE_TIME_OFFSET_ERROR_FOR_IMAGE_TIMESTAMP(50);
 
+constexpr int64_t WAIT_FOR_SYNCHRONIZED_CAPTURE_TIMEOUT = 60000;
+
+#define LOG_LAGGING_TIME(lagger)                                                                                       \
```

Comment:
Just a thought, is there any benefit to using a macro here over a function? We could make `LOG_LAGGING_TIME` and `LOG_SYNCED_IMAGE_TIME` static functions instead of macros. This gives us better type checking, more readable compile errors, and easier stack tracking?